application/back-office/WebApp/shared/translations/locale/nl-NL.po

```diff
@@ -0,0 +1,26 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2024-10-07 23:40+0200\n"
+"PO-Revision-Date: \n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: nl-NL\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: \n"
+"X-Generator: @lingui/cli\n"
+
+msgid "Help"
+msgstr "Helpen"
+
+msgid "Home"
+msgstr "Home"
+
+msgid "Manage tenants, view system data, see exceptions, and perform various tasks for operational and support teams."
+msgstr "Beheer huurders, bekijk systeemgegevens, zie uitzonderingen en voer diverse taken uit voor operationele en ondersteuningsteams."
```

Comment:
I'm sure `huurders` sounds wrong (but if it means someone that rents an apartment it's correct), but remember this is the Backoffice, for internal use.

Reply:
It does sound weird. `Huurders` is indeed correct, if you want the same name as someone who rents an appartement. I'm in doubt though whether `Klanten` isn't a better term in a SaaS application, but that translates to `Customers`.

Reply:
It's the same in all languages. Tenants is also hard for English speaking people. But this is different from Customer, so I'll leave it... it will not be exposed to users.